services/backend/API/admin/reporting/util/countCompetencyProfiles.js

```diff
@@ -21,7 +21,16 @@ const countCompetencyProfiles = async () => {
         COUNT("profiles"."id") DESC;`
   );
 
-  return profileCompetencies[0];
+  const topFiveCompetencies = [];
+
+  profileCompetencies[0].slice(0, 5).forEach(comp => {
+    topFiveCompetencies.push({
+      description: { en: comp.descriptionEn, fr: comp.descriptionFr },
+      count: parseInt(comp.countOccurences) ? parseInt(comp.countOccurences) : 0
+    });
+  });
+
+  return topFiveCompetencies;
```

Comment:
Moving this function into its own file and importing it might be a good idea since it's being used in many places and changing it once can change it everywhere if needed.

Reply:
Fixed it!